ethkey/src/secret.rs

```diff
@@ -55,6 +55,12 @@ impl FromStr for Secret {
 	}
 }
 
+impl From<&'static str> for Secret {
```

Comment:
If it is for tests that might be ok
Maybe just add a comment that it should not be used elsewhere